plugins/auth-backend/src/identity/DatabaseKeyStore.ts

```diff
@@ -62,12 +62,11 @@ export class DatabaseKeyStore implements KeyStore {
 
   async listKeys(): Promise<{ items: StoredKey[] }> {
     const rows = await this.database<Row>(TABLE).select();
-
     return {
       items: rows.map(row => ({
-        key: JSON.parse(row.key),
-        createdAt: utc(row.created_at),
-      })),
+          key: JSON.parse(row.key),
+          createdAt: DateTime.fromFormat(row.created_at, 'yyyy-MM-dd HH:mm:ss', {zone: 'UTC'}),
+        })),
```

Comment:
Hmm, perhaps these are unexpected? Recommend setting `prettier` to run on save in your code editor. :)

Reply:
Normally the prettier githook catches this. Perhaps they didn't install correctly on yarn install? Could try running `yarn husky install` in an attempt to fix it.